tareas/Andres_Sealtiel/Tarea_2.jl

```diff
@@ -0,0 +1,619 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .jl
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Julia 1.6.2
+#     language: julia
+#     name: julia-1.6
+# ---
+
+# <div style="text-align: center"> 
+#
+# # Temas Selectos de Física Computacional I
+# ## Tarea 2
+# #### Ramos Arzate Andres & Pichardo Jiménez, Sealtiel ####
+#
+# ### Ejercicio. 1    
+# </div>
+
+"""
+    iterados(f, x₀, c, n)
+Función que calcula los *'n'* iterados de la función *'f(x,c)'*, evaluada en la condición inicial
+*'x₀'* y con constante *'c'*.
+"""
+function iterados(f, x₀, c, n)
+    iters = zeros(n+1)
```

Comment:
*Sugerencia del código*

Veo que muchos problemas que tuvieron fue con la eficiencia y a la vez parece ser un poco de memoria, si ciertamente estos mensajes mejoren demasiado, pueden llegar a ser claves, no son necesarios que los cambien pero bien pueden probarlos si les interesa.
```suggestion
    iters = Array{Float64}(undef,n+1)
```
Cambiaría por esto por que es más rápido generar un array de valores indefinidos que de 0, en este caso te rellena el array con cualquier valor en vez de seleccionar un valor en especifico. Esto lo hacen muchas veces, pero solo lo comentaré una vez para que no sean muchos comentarios, la recomendación funciona para las veces que se repite esto en otras funciones de la tarea.